test/convert.bats

```diff
@@ -64,3 +64,17 @@ EOF
   rm -f stacker.yaml stacker-subs.yaml
   stacker clean
 }
+
+@test "build python image" {
+  git clone https://github.com/docker-library/python.git
+  chmod -R a+rwx python
+  cd python/3.11/alpine3.17
+  stacker convert --docker-file Dockerfile --output-file stacker.yaml --substitute-file stacker-subs.yaml
+  stacker build -f stacker.yaml --substitute-file stacker-subs.yaml --substitute IMAGE=python
+  if [ -z "${REGISTRY_URL}" ]; then
+      skip "skipping test because no registry found in REGISTRY_URL env variable"
+  fi
+  stacker publish -f stacker.yaml --substitute-file stacker-subs.yaml --substitute IMAGE=python --skip-tls --url docker://${REGISTRY_URL} --tag latest
```

Comment:
My fault for not paying attention, but it seems like 'REGISTRY_URL' should be a URL, not a hostname/path (I'm assuming that a valid setting would be 'docker.io/library', which isn't a url at all).

Reply:
This was borrowed from test/publish.bats

Reply:
.github/workflows/build.yaml
49:          REGISTRY_URL: localhost:5000

Reply:
yeah, we should fix that to be an actual URL, then it would or could support REGISTRY_URL=oci:mirrors/docker.io/library .  I realize you didn't change it here.